librz/main/rizin.c

```diff
@@ -993,8 +993,9 @@ RZ_API int rz_main_rizin(int argc, const char **argv) {
 					if (!haveRarunProfile) {
 						pfile = strdup(argv[opt.ind++]);
 					}
-					perms = RZ_PERM_RX; // XXX. should work with rw too
-					debug = 2;
+					if (strcmp(debugbackend, "winkd")) {
```

Comment:
I recommend adding the comment that this doesn't work with WinKD and why, if possible.